### PR TITLE
[WIP]I589 Update Workspace ->Volume Mount

### PR DESCRIPTION
**Pull request by petra-sargent**
### What does this PR do?

This updates style and grammar.

### What issues does this PR fix or reference?
#589

### CODE REVIEW

**Comment by petra-sargent** [comment]
@rkratky @robin-owen @supriya-bharadwaj  Please review!  Thanks~

**Comment by petra-sargent** [comment]
@preeticp Please review. Thanks!

File: src/main/pages/workspace-admin/volumes.adoc [hunk 1]
```diff
@@ -8,17 +8,19 @@ folder: workspace-admin
...
-[id="user-provided-volumes"]
-== User-Provided Volumes
+Workspace projects are physically located in the `/projects` directory. When a workspace stops, the machines are destroyed, but the volumes persist.
 
-Your workspace may need additional volumes though, say, to persist a local Maven repo, node_modules, ruby jems, authorized_keys for ssh connections etc. You can add additional volumes for your workspace machines - each machine can get as many volumes as an underlying infrastructure can afford, or depending on imposed limits (that’s mostly true for OpenShift).
+[id="Adding-volumes"]
+== Adding volumes
 
-Volumes can be added either in User Dashboard or directly in machine configuration:
+Your workspace may need additional volumes if your data must persist for a local Maven repository, node_modules folder, ruby jems, or authorized_keys file for SSH connections. You can add additional volumes for your workspace machines. Each machine can add as many volumes as the underlying infrastructure can support. OpenShift may impose a limit on the number of volumes.
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/volumes.adoc]: Typos/need styling: `node_modules` directory (not folder), Ruby gems, `authorized_keys` file.

File: src/main/pages/workspace-admin/volumes.adoc [hunk 2]
```diff
@@ -29,18 +31,23 @@ Volumes can be added either in User Dashboard or directly in machine configurati
...
 ----
 
-When adding a volume in User Dashboard using UI or config window, volume name and path are validated to avoid potential failures to create data volumes in Docker and PVCs in OpenShift. If you somehow update workspace configuration using REST API, there can be failures to attach volumes if:
 
-* path containers `~` - an absolute path should be used
-* name and path contains special characters, including `-` and `_`
+To avoid failures when updating the workspace configuration using REST APIs: 
+
+* Be sure to use an absolute path. 
+* The name and path cannot contains special characters, including `-` and `_`.
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/volumes.adoc]: Typo: "contains" should be "contain"

"including `-` or `_`"

File: src/main/pages/workspace-admin/volumes.adoc [hunk 3]
```diff
@@ -8,17 +8,19 @@ folder: workspace-admin
 ---
 
 
-[id="default-volumes"]
-== Default Volumes
+[id="default-volumes_for_workspace_containers"]
+== Default volumes for workspace containers
 
-By default workspace containers/pods start with one default volume/PVC that persists `/projects` where workspace projects are physically located. When a workspace is stopped its machines are destroyed, however, volumes stay there.
+By default, workspace containers start with a default volume and have a minimum of one PVC that is located in `/projects` directory.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/volumes.adoc]:
```suggestion
By default, workspace containers start with a default volume and have a minimum of one PVC that is located in the `/projects` directory. 
```

File: src/main/pages/workspace-admin/volumes.adoc (hunk 1 above)
Comment (rkratky) [reply src/main/pages/workspace-admin/volumes.adoc]: + no 'if'

```suggestion
In order for your data to persist for a local Maven repository, the `node_modules/` directory, Ruby gems, or the `authorized_keys` file for SSH connections, your workspace will need additional volumes. Each machine can add as many volumes as the underlying infrastructure can support. OpenShift may impose a limit on the number of volumes.
```

File: src/main/pages/workspace-admin/volumes.adoc (hunk 2 above)
Comment (rkratky) [inline src/main/pages/workspace-admin/volumes.adoc]:
```suggestion
* The name and path cannot contain special characters, including dashes (`-`) or underscores (`_`).
```

File: src/main/pages/workspace-admin/volumes.adoc [hunk 4]
```diff
@@ -29,18 +31,23 @@ Volumes can be added either in User Dashboard or directly in machine configurati
...
-* name and path contains special characters, including `-` and `_`
+To avoid failures when updating the workspace configuration using REST APIs: 
+
+* Be sure to use an absolute path. 
+* The name and path cannot contains special characters, including `-` and `_`.
 
-If you want your workspace machines to share volumes, just create volumes for each machine with an identical name. This way, machines will share the same volume.
+[NOTE]
+====
+If you want your workspace machines to share volumes, create volumes for each machine that has an identical name. This will allow machines to share the same volume.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/volumes.adoc]: no 'if'

```suggestion
To allow machines to share the same volume, create a volume for each machine with an identical name. 
```

File: src/main/pages/workspace-admin/volumes.adoc [hunk 5]
```diff
@@ -29,18 +31,23 @@ Volumes can be added either in User Dashboard or directly in machine configurati
 }
 ----
 
-When adding a volume in User Dashboard using UI or config window, volume name and path are validated to avoid potential failures to create data volumes in Docker and PVCs in OpenShift. If you somehow update workspace configuration using REST API, there can be failures to attach volumes if:
 
-* path containers `~` - an absolute path should be used
-* name and path contains special characters, including `-` and `_`
+To avoid failures when updating the workspace configuration using REST APIs: 
+
+* Be sure to use an absolute path.
```
Comment (rkratky) [inline src/main/pages/workspace-admin/volumes.adoc]:
```suggestion
* Use an absolute path. 
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Review by rkratky — APPROVED** [review]

**Pull request merged**